src/Entity/IdeasWorkshop/ConsultationReport.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace AppBundle\Entity\IdeasWorkshop;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use AppBundle\Entity\EntityNameTrait;
+use AppBundle\Entity\EntityPositionTrait;
+use AppBundle\Entity\EntityUrlTrait;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class ConsultationReport
+{
+    use EntityNameTrait;
+    use EntityUrlTrait;
+    use EntityPositionTrait;
+
+    /**
+     * @ORM\Column(type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
```

Comment:
Same here, you can just use directly : `@ORM\GeneratedValue`

Reply:
:heavy_check_mark: